.azdevops/image-pipeline.yml

```diff
@@ -7,11 +7,8 @@ pr:
 - master
 variables:
 - group: 'Metrics Adapter'
-# This value is set for convenience of re-tagging the image later, but is otherwise irrelevant
-- name: 'REGISTRY_PATH'
-  value: ''
 jobs:
-- job:
+- job: Image

```

Comment:
Is this the name of the job? I think this would make more sense as "Build and Test" or something similiar